### ISSUE TITLE
Support for latest npm

### ISSUE THREAD

**Issue by mrac** [zalando/dress-code#144]
It looks like dresscode supports old **npm** dependencies model.

The latest **npm** (version 3) has different dependency model than **npm 2**:
https://docs.npmjs.com/how-npm-works/npm3

The relative import paths to breakpoint/susy cause error during sass compilation, because in latest **npm** all sub-dependencies are installed in root ```node_modules``` directory.

This is currently a blocker for our app to deploy dresscode-integration to production, because we use latest npm ver.3 and node ver.5.

```npm install``` gives:

```
npm WARN engine dress-code@1.1.1-alpha: wanted: {"node":">=0.10.0 <=4.x","npm":"^2.0.0"} (current: {"node":"5.5.0","npm":"3.3.12"})
```

SASS compilation gives:

```
Message:
    node_modules/dress-code/dist/sass/core/_mixins.scss
Error: File to import not found or unreadable: ../../../node_modules/breakpoint-sass/stylesheets/breakpoint
       Parent style sheet: /Users/mraczkowski/Work/repos/BRAND/brands-frontend-analytics/node_modules/dress-code/dist/sass/core/_mixins.scss
        on line 2 of node_modules/dress-code/dist/sass/core/_mixins.scss
>> @import "../../../node_modules/breakpoint-sass/stylesheets/breakpoint";
   ^

Details:
    formatted: Error: File to import not found or unreadable: ../../../node_modules/breakpoint-sass/stylesheets/breakpoint
       Parent style sheet: /Users/mraczkowski/Work/repos/BRAND/brands-frontend-analytics/node_modules/dress-code/dist/sass/core/_mixins.scss
        on line 2 of node_modules/dress-code/dist/sass/core/_mixins.scss
>> @import "../../../node_modules/breakpoint-sass/stylesheets/breakpoint";
   ^

    column: 1
    line: 2
    file: /Users/mraczkowski/Work/repos/BRAND/brands-frontend-analytics/node_modules/dress-code/dist/sass/core/_mixins.scss
    status: 1
    messageFormatted: node_modules/dress-code/dist/sass/core/_mixins.scss
Error: File to import not found or unreadable: ../../../node_modules/breakpoint-sass/stylesheets/breakpoint
       Parent style sheet: /Users/mraczkowski/Work/repos/BRAND/brands-frontend-analytics/node_modules/dress-code/dist/sass/core/_mixins.scss
        on line 2 of node_modules/dress-code/dist/sass/core/_mixins.scss
>> @import "../../../node_modules/breakpoint-sass/stylesheets/breakpoint";
   ^

    messageOriginal: File to import not found or unreadable: ../../../node_modules/breakpoint-sass/stylesheets/breakpoint
Parent style sheet: /Users/mraczkowski/Work/repos/BRAND/brands-frontend-analytics/node_modules/dress-code/dist/sass/core/_mixins.scss
    relativePath: node_modules/dress-code/dist/sass/core/_mixins.scss
```

**Comment by rbarilani** [comment]
Hi @mrac. Yep, we are aware of this issue. See current master [README - how to use dress code sass](https://github.com/zalando/dress-code/tree/master#how-to-use-dress-code-sass).  
This constraint will be removed in the next upcoming release... we already merged that change in the develop branch. See https://github.com/zalando/dress-code/issues/129.

With the next release, hopefully 1.0.0-beta, we will introduce a quiet amount of breaking changes for certain components. So before upgrading always check the changelog and test the new version.

Maybe we can do an intermediate release before going beta... I will check that with the team. Is that ok for you?

**Comment by mrac** [comment]
Hey @rbarilani! Thanks! The intermediate release would be great, or at least a quick fix to point to root node_modules dependency (instead of nested dependency).

**Comment by rbarilani** [comment]
closed with release 1.2.0
